profiles/car.lua

```diff
@@ -544,12 +544,13 @@ function way_function (way, result)
 end
 
 function turn_function (angle)
-  ---- compute turn penalty as angle^2, with a left/right bias
+  -- Use a sigmoid function to return a penalty that maxes out at turn_penalty
+  -- over the space of 0-180 degrees.  Values here were chosen by fitting
```

Comment:
Does this mean this function expecting input from `0` - `180`? How does that reconcile with [this comment](https://github.com/Project-OSRM/osrm-backend/blob/b19a5cc0201f0c57532756d6b0852cf6f20e05dc/include/util/guidance/toolkit.hpp#L32-L35) about how turn angles range from `0` - `360`?

Also, [this comment](https://github.com/Project-OSRM/osrm-backend/issues/2918#issuecomment-247380039) implies that the turn function is passed negative values (e.g. `turn_function(-135)`).

In short, what is the range of possible values that can be passed to `turn_function()`?